src/main/kotlin/fi/metatavu/timebank/api/severa/models/SeveraWorkContract.kt

```diff
@@ -0,0 +1,17 @@
+package fi.metatavu.timebank.api.severa.models
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.quarkus.runtime.annotations.RegisterForReflection
+
+/**
+ * Data class for workContract data coming from Severa
+ */
+@RegisterForReflection
+@JsonIgnoreProperties(ignoreUnknown = true)
+class SeveraWorkContract {
+    @JsonProperty("startDate")
```

Comment:
You don't need @JsonProperty if the JSON field has the exact same name as the POJO does.